Meta/.shell_include.sh

```diff
@@ -3,7 +3,10 @@
 # SC2034: "Variable appears unused. Verify it or export it."
 #         Those are intentional here, as the file is meant to be included elsewhere.
 
-SUDO="sudo"
+# NOTE: If using another privillege escalation binary make sure it is configured or has the appropiate flag
+# 	to keep the current environment variables in the launched process (in sudo's case this is achieved
+# 	through the -E flag described in SUDO(8).
+SUDO="sudo -E"
 
 if [ "$(uname -s)" = "SerenityOS" ]; then
     SUDO="pls"
```

Comment:
In that case, please add the `-E`/`--preserve-env` flag to the `pls` variant as well.